### DISCUSSION TITLE
Run cell2location with multiple Visium slides or one at a time

### DISCUSSION THREAD

**Discussion by kuang-da** [BayraktarLab/cell2location#295] [Q&A] [answered]
Hi there,

Firstly, I would like to express my gratitude for the exceptional work that has gone into developing Cell2location. This computation solution for spatial transcriptomics has greatly impressed me, and I am currently applying it to my own dataset. I'd greatly appreciate some guidance in this process.

I'm working with 16 slides sourced from 4 regions of a single organ, obtained from two different donors. For clarification, my samples are labeled as follows:

```
- Region-a-Donor-x-Slide-1
- Region-a-Donor-x-Slide-2
- Region-a-Donor-y-Slide-1
- Region-a-Donor-y-Slide-2
```
Where the variable a can be replaced with b, c, and d.

My reference dataset consists of 16-paired scRNA-seq data. I've followed the steps outlined in the [QUICK START TUTORIAL](https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html) and have successfully run Cell2location to deconvolve each individual Visium slide based on this reference.

I have noticed that the Cell2location model also supports the processing of multiple batches of Visium slides simultaneously. I presume that under this setup, the experiment specific $s_{e,g}$, which encapsulates the gene-specific additive shift, would be estimated.

Considering this, I have two questions:

- Is there a benefit to processing multiple slides simultaneously as opposed to individually?
- If I take separated deconvolved Visium slides together and apply KNN+ Leiden clustering, would the region identities still be directly comparable?

Your insights on this matter would be highly valuable for my ongoing work. Thank you in advance for your time and support.

**Comment by vitkl** [comment, marked as answer]
Hi @kuang-da 

A joint analysis of all sections is useful because it corrects a number of technical effects and leverages more data to estimate and distinguish cell abundance and technology difference effect (greater sensitivity).

We recommend using a joint scRNA reference from all sections and joint analysis of all Visium sections - unless there are good reasons to do other wise such as when modelling cancer data from different patients.

**Reply by pedrocastillorosique** [reply]
Hi! Sorry, but I was wondering if having four different samples from same cancer type would be beneficious? Thank you

**Comment by kuang-da** [comment]
Thank you for the feedback! I am rerunning the analysis with all Visium slides concatenated.
